docs/manual/config/keys.rst

```diff
@@ -45,6 +45,82 @@ The :class:`EzKey` modifier keys (i.e. ``MASC``) can be overwritten through the
        'C': 'control',
     }
 
+KeyChords
+=========
+
+Qtile also allows sequences of keys to trigger callbacks. In Qtile, these
+sequences are known as chords and are defined with
+:class:`libqtile.config.KeyChord`. Chords are added to the ``keys`` section of
+the config file.
+
+::
+
+    from libqtile.config import Key, KeyChord
+
+    keys = [
+        KeyChord([mod], "z", [
+            Key([], "x", lazy.spawn("xterm"))
+        ])
+    ]
+
+The above code will launch xterm when the user presses Mod + z, followed by x.
+
+.. warning::
+    Users should note that key chords are aborted by pressing <escape>. In the
+    above example, if the user presses Mod + z, any following key presses will
+    still be sent to the currently focussed window. If <escape> has not been
+    pressed, the next press of x will launch xterm.
+
+Modes
+-----
+
+Chords can optionally specify a "mode". When this is done, the mode will remain
+active until the user presses <escape>. This can be useful for configuring a
+subset of commands for a particular situations (i.e. similar to vim modes).
+
+::
+
+    from libqtile.config import Key, KeyChord
+
+    keys = [
+        KeyChord([mod], "z", [
+            Key([], "plus", lazy.layout.grow()),
+            Key([], "minus", lazy.layout.shrink()),
+            Key([], "n", lazy.layout.normalize()),
+            Key([], "m", lazy.layout.maximize())],
+            mode="Windows"
+        )
+    ]
+
+In the above example, pressing Mod + z triggers the "Windows" mode. Users can
+then resize windows by just pressing g (to grow the window), s to
```

Comment:
I think `g` and `s` here are meant to be `plus` and `minus`, right?

Reply:
Sigh... Thought I'd fixed that before pushing. The plus didn't work when testing the code so I changed to "'s" and "g". Will fix.